plugins/outputs/datadog/README.md

```diff
@@ -36,6 +36,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Override the default (none) compression used to send data.
   ## Supports: "zlib", "none"
   # compression = "none"
+
+  ## Convert counts to rates
+  ## Use this to be able to submit metrics from telegraf alongside Datadog agent
+  # should_rate_counts = true
+
+  ## When should_rate_counts is enabled, this overrides the
+  ## default (10s) rate interval used to divide count metrics by
+  # rate_interval = 20
```

Comment:
The example config and readme should show the default values. In this case `false` and `10s`. That way you don't have to document the default in the help text.

I do think we should get rid of `should_rate_counts`, and use a non-zero rate_interval, or like we do in stackdriver output, we have a `metric_counter = []` which means any metric matching that name will be marked as a counter. Should we do the same here and mark any metric matching that name as a rate?

Reply:
Appreciate the feedback.

> we have a metric_counter = [] which means any metric matching that name will be marked as a counter. Should we do the same here and mark any metric matching that name as a rate?

It's a good call out, but that would not scale in larger deployments such as the one I am working with that has 1000s of metrics. 

I am happy with exploring having `rate_interval` default to 0, and if it is > 0 then enable conversion of counts to rates. My concern would be if users would know to set it to 10 - which is the default rate interval used by the Datadog agent - I can explicitly state this in the README though.

If it were to stay then I have no objections to changing `should_rate_counts` to `convert_count_to_rate`.

Let me know your thoughts 👍🏻

Reply:
I'd still like to see the single option and you can document the suggested interval would be 10 seconds as it is default used by Datadog. Sound good?

Reply:
SGTM - I've made the change to use single option in https://github.com/influxdata/telegraf/pull/15702/commits/c80846351828646a971d5763485fb965e6e1784b.